plugins/tiddlywiki/dynannotate/examples/viewtemplate.tid

```diff
@@ -16,13 +16,12 @@ Once enabled, this example demonstrates several features:
 </div>
 
 <$button>
-<$action-setfield $tiddler="$:/core/ui/ViewTemplate/body" $field="text" $value={{$:/plugins/tiddlywiki/dynannotate/examples/viewtemplate/text}}/>
-<$action-setfield $tiddler="$:/core/ui/ViewTemplate/body" $field="tags" $value="$:/tags/ViewTemplate"/>
+<$action-setfield $tiddler="$:/config/ViewTemplateTitleFilters/fish" tags="$:/tags/ViewTemplateTitleFilter" text="[[$:/plugins/tiddlywiki/dynannotate/examples/viewtemplate/text]]" list-before="$:/config/ViewTemplateBodyFilters/default"/>
 Click here to use the custom view template
 </$button>
 
 <$button>
-<$action-deletetiddler $tiddler="$:/core/ui/ViewTemplate/body"/>
+<$action-deletetiddler $tiddler="$:/config/ViewTemplateTitleFilters/fish"/>
```

Comment:
Where does this config come from? IMO it shouldn't be there at all.

Reply:
Sorry. I forgot to change the name in the test case. I will correct it.

Reply:
> IMO it shouldn't be there at all.

I have fixed it to: `$:/config/ViewTemplateTitleFilters/dynannotate`